libgraph/src/BuildGraph.cpp

```diff
@@ -1895,7 +1896,11 @@ katana::WritePropertyGraph(
         "edge prop: ({}) {}", field->type()->ToString(), field->name());
   }
 
-  auto result = prop_graph.Write(dir, "libkatana_graph");
+  // Since this function is only used by tools under
+  // external/katana/tools/graph-convert/, but not any other graph
```

Comment:
You should check with @patrickkenney9801 to see if these tools are still in use.

Reply:
I don't think that tooling is still in use, I think @tylershunt exposed graphml importing from another API as well